lib/simplecov/configuration.rb

```diff
@@ -288,6 +290,10 @@ def add_group(group_name, filter_argument = nil, &filter_proc)
 
   private
 
+    def minimum_possible_coverage_exceeded(method_name)
+      warn "The coverage you set for #{method_name} is greater than 100%"
```

Comment:
I think we should rather call this `coverage_option` or something (still doesn't sound great) - _method_name_ makes me think we're doing fancy metaprogramming stuff :)